Automattic/wp-calypso
# Stats: Update Post Detail view components directories

This PR moves
- `post.jsx`
- `module-post-months.jsx`
- `module-post-weeks.jsx`

into a `stats/post-detail` directory and changes the names a little:
- `post.jsx` is now `post-detail/index.jsx`
- `module-post-months.jsx` is now `detail-months.jsx`
- `module-post-weeks.jsx` is now `detail-weeks.jsx`

__To Test__
- Open up a stats page for a time period ( month/week/day )
- Click on a post in the "Post & Pages" component
- Verify the subsequent post detail page looks the same as it does on production
